src/newViews/Tagging/mockData.ts

```diff
@@ -0,0 +1,154 @@
+import { Entry } from './types';
```

Comment:
We should verify that the the entry information is conformant with analysis framework.
We should either wait for the AF mock data or fix this after it is merged